pytest_fixtures/component/contentview.py

```diff
@@ -68,3 +68,15 @@ def module_cv_repo(module_org, module_repository, module_lce, module_target_sat)
     content_view = content_view.read()
     content_view.version[0].promote(data={'environment_ids': module_lce.id, 'force': False})
     return content_view
+
+
+@pytest.fixture(scope='session')
+def session_multicv_sat(session_satellite_host):
+    """Satellite with multi-CV enabled"""
```

Comment:
Are you going to add tests related to this fixture later or missed pushing?

Reply:
Yes, @jeremylenz is about to add a few based on this PR.